.github/workflows/ci.yml

```diff
@@ -10,6 +13,9 @@ jobs:
   linters:
     name: Linters
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
I think this is not needed. At least it seems to work in my fork that doesn't have it 🤔